specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/webapplicationfirewall.json

```diff
@@ -453,13 +453,28 @@
           "type": "boolean",
           "description": "Whether to allow WAF to check request Body."
         },
+        "reqBodyInspectLimitInKB": {
```

Comment:
nit: maxRequestBodySizeInKb and fileUploadLimitInMb have small 'b'. Is this intentional?

Reply:
@sindhualuguvelli1 this is from before. We dont want to make the same mistakes as done in the past.

Reply:
"B" represents Bytes whereas "b" means bits. So please ensure we are actually treating this as Bytes and not Bits.

Reply:
yes we are treating this as Bytes. We have confirmed this in the code.